src/etc/inc/plugins.inc.d/dpinger.inc

```diff
@@ -320,7 +320,7 @@ function dpinger_status()
         $status[$gwname] = $report = [
             'status' => !empty($gwitem['force_down']) ? 'force_down' : $gwstatus,
             /* grab the runtime monitor from the instance if available */
-            'monitor' => $gwitem['monitor'] ?? '~',
+            'monitor' => !empty($gwitem['monitor']) ? $gwitem['monitor'] : '~',
```

Comment:
don't mind moving this to master so we don't have to though dpinger.inc from this PR again :)

Reply:
done :)